lib/parser/v1.ts

```diff
@@ -1,8 +1,29 @@
+import {
+  PathObj,
+  Policy,
+  PolicyError,
+  Rule,
+  RuleSet,
+  Severity,
+} from '../types';
+
```

Comment:
Is it possible that `PolicyError` does not exist in `../types`?

Reply:
Oh, yes, it wasn't committed... why doesn't the TS transpiler catch this??? :|